.github/workflows/github-release.yml

```diff
@@ -0,0 +1,57 @@
+name: github-release
+on:
+  push:
+    tags:
+    - "v[0-9]+.[0-9]+.[0-9]+"
+jobs:
+  github-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        # We neeed to fetch the entire history as conventional-changelog needs
+        # access to any number of git commits to build the changelog.
+        with:
+          fetch-depth: 0
+
+      - name: read node version from .nvmrc
+        run: echo ::set-output name=NVMRC::$(cat .nvmrc)
+        shell: bash
+        id: nvm
+
+      - name: setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '${{steps.nvm.outputs.NVMRC}}'
+          cache: npm
+
+      - name: npm install
+        run: npm i --prefer-offline --no-audit
+
+      - name: build
+        run: npm run build-prod
+
+      - name: get all dist files
+        run: echo ::set-output name=DISTFILES::$(find dist -type f -name '*\.js' -or -name '*\.css')
+        shell: bash
+        id: dist-files
+
+      - name: get dashed package version for netlify
+        run: echo ::set-output name=VERSION::$(node -e "process.stdout.write(require('./package.json').version.split('.').join('-'))")
+        id: get-version
+        shell: bash
+
+      - name: Generate current release changelog
+        run: ./node_modules/.bin/conventional-changelog -p videojs -r 2 -o RELEASE-CHANGELOG.md
+
+      - name: add netlify preview to release notes
+        run: |
+          echo "" >> RELEASE-CHANGELOG.md
+          echo "[netlify preview](https://v${{steps.get-version.outputs.VERSION}--videojs-http-streaming.netlify.app)" >> RELEASE-CHANGELOG.md
```

Comment:
cool idea!
Does this get sluggified properly? Since the preview url is something like https://v2-10-0--videojs-http-streaming.netlify.app/ rather than https://v2.10.0--videojs-http-streaming.netlify.app/

Reply:
It does, i convert it to a dashed version in the `node -e` command above via `split('.').join('-')`

Reply:
ah, cool, missed that piece.